en/prototyping/workspaces.md

```diff
@@ -84,4 +84,4 @@ There's a new participant in a workspace usability study | Subscribe to notifica
 [a-4]: #managing-group-workspaces
 [a-5]: #configuring-notifications
 
-[topic-1]: setting-up-a-usability-study.md
+[topic-1]: set-up-a-user-test.md
```

Comment:
Change  [Setting Up a Usability Study][topic-1] to * [Setting Up a User Test][topic-1]

and remove the a-1 to a-5 references as they are not used in the topic above

Reply:
@ig-georgeA any comments here?

Reply:
@StefanIvanov thanks for waiting. Getting familiar with the github workflows. :)

Reply:
I think this is not resolved yet:
Change [Setting Up a Usability Study][topic-1] to * [Setting Up a User Test][topic-1] and remove the a-1 to a-5 references as they are not used in the topic above